deploy/kubernetes/rbac.yaml

```diff
@@ -51,6 +51,14 @@ rules:
     resources: ["customresourcedefinitions"]
     verbs: ["create", "list", "watch", "delete", "get", "update"]
 
+#Secret permission is optional.
+#Enable it if you need value from secret.
+#For example, you have key `csi.storage.k8s.io/controller-publish-secret-name` in StorageClass.parameters.
```

Comment:
For example, `csi.storage.k8s.io/snapshotter-secret-name` is set in VolumeSnapshotClass.

Reply:
fixed